src/main/java/org/broadinstitute/hellbender/tools/exome/allelefraction/AFParameters.java

```diff
@@ -0,0 +1,69 @@
+package org.broadinstitute.hellbender.tools.exome.allelefraction;
+
+/**
+ * Simple container for the allele fraction model's parameters at a single het site:
```

Comment:
"single het" or "single segment"?

it seems like this class unnecessarily duplicates much of the info in AFState. i think it might be cleaner to pass the full state + index of the desired segment for methods that require per-segment parameters. maybe @LeeTL1220 can chime in.